lib/ansible/plugins/strategy/__init__.py

```diff
@@ -1171,6 +1170,8 @@ def _evaluate_conditional(h):
                     if host.name not in self._tqm._unreachable_hosts:
                         iterator._host_states[host.name].run_state = iterator.ITERATING_COMPLETE
                 msg = "ending play"
+            else:
+                skipped = True
```

Comment:
we should add skipped reason as we do in tasks when conditional forces skip

Reply:
I'll work on adding that to this and the other meta ones in a separate PR.